### PR TITLE
Adds cubic-bezier() and steps() easing functions

### PR DESCRIPTION
**Pull request by jamesnw**

### CODE REVIEW

File: features/cubic-bezier-easing.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: cubic-bezier() easing
+description: The `cubic-bezier()` easing function interpolates from 0-1 along a smooth curve influenced by two control points, used to create animations and transitions with continuous changes in speed. The `ease`, `ease-in`, `ease-out`, and `ease-in-out` CSS properties are presets for common scenarios.
```
Comment (jamesnw) [inline features/cubic-bezier-easing.yml]:
```suggestion
description: The `cubic-bezier()` easing function creates animations and transitions with continuous changes in speed. The `ease`, `ease-in`, `ease-out`, and `ease-in-out` CSS properties are presets for common scenarios.
```
Potentially, we could add `interpolates from 0-1 along a smooth curve influenced by two control points` to the end, but I'm not sure if it's a crucial part- I think the "continuous changes in speed" line explains it better (if less specifically).

File: features/cubic-bezier-easing.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+spec: https://drafts.csswg.org/css-easing-2/#cubic-bezier-easing-functions
+group: css
+compat_features:
+  - css.types.easing-function.cubic-bezier
```
Comment (jamesnw) [inline features/cubic-bezier-easing.yml:6]: I'm surprised that ease-in, etc, don't have keys.

File: features/step-easing.yml
Comment (jamesnw) [inline features/step-easing.yml:1]: I think the file name should be `steps-easing` to match the name and property

File: features/step-easing.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+name: steps() easing
+description: The `steps()` easing function divides an animation or transition into equidistant steps. The `step-start` and `step-end` CSS properties are presets for specific scenarios.
```
Comment (jamesnw) [inline features/step-easing.yml]:
```suggestion
The `steps()` easing function breaks an animation or transition into a number of equally spaced intervals, creating a segmented or jumpy effect. The `step-start` and `step-end` CSS properties are presets with a single step.
```

Attempt at making the definition non-circular, not sure I love it.

**Review by jamesnw — COMMENTED** [review]
@vwallen I opened a PR so I could give some feedback. Thanks!

File: features/cubic-bezier-easing.yml (hunk 1 above)
Comment (vwallen) [reply features/cubic-bezier-easing.yml]: I was following the pattern established by the `linear()` easing function here:
> The `linear()` easing function for animations and transitions interpolates linearly between the control points, and can be used to approximate complex easing functions, such as a bounce effect.

I'm not really a fan of "creates" here, but fewer internal details would probably help. Maybe something more like: 
> The `cubic-bezier()` easing function interpolates along a smooth curve, creating animations and transitions with continuous changes in speed. The `ease`, `ease-in`, `ease-out`, and `ease-in-out` CSS properties are presets for common scenarios.

Comment (jamesnw) [reply features/cubic-bezier-easing.yml]: I like your proposal! I think you can drop the word "used" here.

File: features/step-easing.yml (hunk 3 above)
Comment (vwallen) [reply features/step-easing.yml]: How about:
> The `steps()` easing function divides an animation or transition into equidistant intervals, jumping from value to value. The `step-start` and `step-end` CSS properties are presets with a single interval.

Comment (jamesnw) [reply features/step-easing.yml]: Looks good to me!

**Pull request closed without merging**